tools-local/tasks/mobile.tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -187,7 +187,7 @@ void EmitNativeToInterp(StreamWriter w, List<PInvokeCallback> callbacks)
         foreach (var cb in callbacks) {
             var method = cb.Method;
 
-            if (method.ReturnType != typeof(void) && !IsBlittable(method.ReturnType))
+            if (method.ReturnType.Name != "Void" && !IsBlittable(method.ReturnType))
```

Comment:
~`"Void"`~ -> `typeof(void).Name`?

Reply:
The other places in this file are comparing with `"Void"`, so I went with that for consistency. Ideally, this should really check the full namespace and assembly identity, but it is just a non-shipping helper script so not worth it spending much time on it.